salvo/src/lib/docker/BUILD

```diff
@@ -0,0 +1,39 @@
+load("@rules_python//python:defs.bzl", "py_library", "py_test")
+
+licenses(["notice"])  # Apache 2
+
+package(
+    default_visibility = ["//:__subpackages__"],
+)
+
+py_library(
+    name = "docker_image",
+    data = [
```

Comment:
Looks like I completely missed this before. I believe this shouldn't be marked as `data =`, but as `srcs =`. There is a difference, `srcs` tells blaze that these are code source files while `data` instructs blaze to package these files into the generated runfiles directory.

https://docs.bazel.build/versions/master/be/python.html#py_library

If you don't mind, let's clean this up in a separate PR.

Reply:
Sure thing.  Not sure how this one escaped me either.